src/Docker/Runner/DataLoader/DataLoader.php

```diff
@@ -154,7 +154,8 @@ public function loadInputData(InputTableStateList $inputTableStateList)
                 $this->logger->debug('Downloading source files.');
                 $reader->downloadFiles(
                     $this->storageConfig['input']['files'],
-                    $this->dataDirectory . DIRECTORY_SEPARATOR . 'in' . DIRECTORY_SEPARATOR . 'files',
+                    $this->dataDirectory, // obsahuje i /data/
+                    '/data/' . DIRECTORY_SEPARATOR . 'in' . DIRECTORY_SEPARATOR . 'files',
```

Comment:
Preco je tu to `/data/` ked pises, ze $dataDirectory uz to obsahuje? Akoze to bude /data/data/.. ?

Reply:
to je uplnej bullshit a vubec to sem nepatri :)